packages/okta-adapter/test/filters/service_url.test.ts

```diff
@@ -0,0 +1,115 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { filterUtils } from '@salto-io/adapter-components'
+import { CORE_ANNOTATIONS, ElemID, InstanceElement, ObjectType, ReferenceExpression, getChangeData, toChange } from '@salto-io/adapter-api'
+import { getFilterParams, mockClient } from '../utils'
+import OktaClient from '../../src/client/client'
+import serviceUrlFilterCreator from '../../src/filters/service_url'
+import { APPLICATION_TYPE_NAME, OKTA, USERTYPE_TYPE_NAME, USER_SCHEMA_TYPE_NAME } from '../../src/constants'
+
+describe('serviceUrlFilterCreator', () => {
+  let client: OktaClient
+    type FilterType = filterUtils.FilterWith<'onFetch' | 'onDeploy'>
+    let filter: FilterType
+
+    beforeEach(async () => {
+      jest.clearAllMocks()
+      const mockCli = mockClient()
+      client = mockCli.client
+      filter = serviceUrlFilterCreator(getFilterParams({ client })) as typeof filter
+    })
+
+    describe('cheking object types that use addUrlToInstance() and can be filtered using the config', () => {
```

Comment:
```suggestion
    describe('checking object types that use addUrlToInstance() and can be filtered using the config', () => {
```

tiny nit: it can be shortened to `types using serviceUrl config`